sycl/test/abi/layout_exception.cpp

```diff
@@ -29,7 +29,7 @@ void foo() {
 // CHECK-NEXT:        32 |       element_type * _M_ptr
 // CHECK-NEXT:        40 |       class std::__shared_count<> _M_refcount
 // CHECK-NEXT:        40 |         _Sp_counted_base<(_Lock_policy)2U> * _M_pi
-// CHECK-NEXT:        48 |   class std::error_code MErrC
+// CHECK-NEXT:                     std::error_code MErrC
```

Comment:
Can we instead do something like `{{class|struct}}` std::error_code... ?

Reply:
We can, but I don't think that will improve the testing, eg: we may fail again if there are some other keywords around it due to different env.

Reply:
I think "48" is kind of important here because we do want to know the offsets and ensure it's unchanged for the purposes of ABI compatibility. I think @uditagarwal97 's initial suggestion is the way to go.

Reply:
Fine for me for now.